config/config.exs

```diff
@@ -17,6 +17,10 @@ config :exile_web,
   ecto_repos: [Exile.Repo],
   generators: [context_app: :exile]
 
+config :exile_auth,
+  ecto_repos: [Exile.Repo],
```

Comment:
A REPO should be owned by the OTP App using it, it should not be in the `exile` app